Dockerfile

```diff
@@ -22,25 +26,27 @@ COPY valheim-* /usr/local/bin/
 COPY defaults /usr/local/etc/valheim/
 COPY common /usr/local/etc/valheim/
 COPY contrib/* /usr/local/share/valheim/contrib/
-RUN if [ "${TESTS:-true}" = true ]; then shellcheck -a -x -s bash -e SC2034 /usr/local/sbin/bootstrap /usr/local/bin/valheim-* /usr/local/share/valheim/contrib/*.sh; fi
-RUN mkdir -p \
-        /artifacts/bin \
-        /artifacts/tmp
-WORKDIR /artifacts
-RUN mv /build/busybox/_install/bin/busybox ./bin/busybox
-RUN mv /build/vpenvconf/dist/vpenvconf-*.linux-x86_64.tar.gz ./tmp/vpenvconf.tar.gz
-RUN mv /build/python-a2s/dist/python-a2s-*.linux-x86_64.tar.gz ./tmp/python-a2s.tar.gz
-COPY bootstrap ./sbin/
-COPY status-updater.py ./bin/
-COPY valheim-* ./bin/
-COPY defaults ./etc/valheim/
-COPY common ./etc/valheim/
-COPY contrib/* ./share/valheim/contrib/
+RUN if [ "${TESTS:-true}" = true ]; then \
+        shellcheck -a -x -s bash -e SC2034 \
+            /usr/local/sbin/bootstrap \
+            /usr/local/bin/valheim-backup \
+            /usr/local/bin/valheim-bootstrap \
+            /usr/local/bin/valheim-server \
+            /usr/local/bin/valheim-updater \
+            /usr/local/bin/valheim-plus-updater \
+            /usr/local/share/valheim/contrib/*.sh \
+        ; \
+    fi
+RUN mkdir -p /usr/local/tmp
+WORKDIR /
+RUN mv /build/busybox/_install/bin/busybox /usr/local/bin/busybox
+RUN tar xzvf /build/vpenvconf/dist/vpenvconf-*.linux-x86_64.tar.gz
+RUN tar xzvf /build/python-a2s/dist/python-a2s-*.linux-x86_64.tar.gz
```

Comment:
What is the purpose of these two lines?  Nothing resulting from this ever gets copied to the second build stage.  I think it's best to copy the tarballs into `/usr/local` (or whichever directory we are copying from in the second stage) and then extract them there.

Reply:
What do you mean? These extract directly to `/usr/local` like they did in the main container. Just that they do it now in the build one. See the build output.

Reply:
Ohhhh you're right, missed that.